information-sources/overview-information-sources.md

```diff
@@ -0,0 +1,18 @@
+---
+slug: /posdealers/overview/overview-information-sources
+title: Overview Information Sources
+---
+
+# Overview Information Sources
+
+:::info summary
+
+After reading, PosDealers can use KBA, track cases and review partner status. You get details on news, videos, webinars, support, and how to use and create this documentation.
```

Comment:
when I compare it to the section in overview getting started maybe:
After reading **this**, **you** can...